fastlane/Fastfile

```diff
@@ -219,14 +219,10 @@ platform :ios do
       release_notes_file_path: RELEASE_NOTES_SOURCE_PATH
     )
 
-    if prompt_for_confirmation(
-      message: 'Ready to push changes to remote and let the automation configure the release on GitHub?',
-      bypass: ENV.fetch('RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM', nil)
-    )
-      push_to_git_remote(tags: false)
-    else
-      UI.message('Aborting code completion. See you later 👋')
-    end
+    UI.important('Pushing changes to remote and configuring the release on GitHub')
+    UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
```

Comment:
Check out @oguzkocer comment about this wording in the WordPress counterpart, https://github.com/wordpress-mobile/WordPress-iOS/pull/22191#discussion_r1424460296 and my tentative rewording https://github.com/wordpress-mobile/WordPress-iOS/commit/919d62c677240eadebc6a2602061ffcf870b0de6

What do you think?

Also, I too decided to remove the "See you later" bit 😄

Reply:
I like that new wording. Updated in https://github.com/woocommerce/woocommerce-ios/commit/8e63f48ea54a5283d132c10618837975936507de